### ISSUE TITLE
Who is the audience for our output? 

### ISSUE THREAD

**Issue by RubenVerborgh** [solid/Explaining-the-Vision-Panel#2]

**Comment by RubenVerborgh** [comment]
My take: general public. As in: people, 14 years and up, at least medium proficiency in English (not necessarily native speakers), not necessary technical.

**Comment by danwilkinsoncreative** [comment]
I'd agree that these statements should be as accessible as possible although perhaps without stretching/diluting them so much as to render them useless to the people most relevant right now e.g. developers, journalists, business owners, government officials etc

**Comment by pjworrall** [comment]
I agree with that audience. The important distinction for me is that it is not developer community.

**Comment by danwilkinsoncreative** [comment]
Key things for us to consider with such a broad audience is how we not only make it clear and understandable to all but also how we make it relevant and engaging. What common needs/desires/motives do they all share and how can we satisfy those with these statements?
